.github/workflows/regression-tests.yml

```diff
@@ -38,23 +38,25 @@ jobs:
       CYPRESS_LOGIN_TEST_OTP_SECRET_AUDITEE: ${{ secrets.CYPRESS_LOGIN_TEST_OTP_SECRET_AUDITEE }}
       DISABLE_AUTH: False
       # useful for debugging but otherwise complete overwhelm
-      # DEBUG: cypress:*
+      # DEBUG: cypress:server:util:process_profiler
     steps:
       - name: Checkout
         uses: actions/checkout@v4
 
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+
       - name: Create .env file
         run: touch .env
 
       - name: Run Cypress full submission test
-        uses: Wandalen/wretry.action@master
```

Comment:
Keep in mind, this action was added to assist with the 60000ms timeout errors that persisted through the cypress workflow. If that is no longer an issue, it can be removed, but this is just a loop that runs 5 times.

Reply:
I think the retries may have been hiding some of the underlying flakiness in our e2e tests, which has now gotten to the point where even retrying 5 times doesn't seem to help. 

I initially removed the retry action so I could test a Node compatibility issue: because the cypress tests are nested under `wretry.action`, they're kicked off using wretry's version of Node (16), which also installs npm dependencies. But the _cypress_ action is hardcoded to use Node 20, which is unfortunate and resulted in some missing dependency errors in the logs.

Anyway, consistifying all of the Node versions does seem to have resolved some warnings, but the tests are still flaky in new and exciting ways.

Short term, I think we should just simplify the cypress test that confirms files were uploaded: catch the 302 response from the server, but don't bother checking for corresponding page updates.

Medium term, we need to refactor the Cypress tests to speed them up and make sure each test is actually testing what we want. Cypress also supports more granular retry functionality directly, but I'm not sure it would work well with the current full_submission spec.

Reply:
Yup! Makes sense to me. If we aren't seeing the timeouts we were, then get rid of it. Thanks for the explanation.